demos/connector/neo4j/load-cora-into-neo4j.ipynb

```diff
@@ -470,12 +470,28 @@
     "    tx.evaluate(loading_edge_query, parameters={\"edge_list\": records})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Ensure node IDs are unique. Creating this constraint also automatically creates an index which will improve performance of querying nodes by ID."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": 14,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "node_id_constraint = \"\"\"\n",
+    "    CREATE CONSTRAINT\n",
```

Comment:
I wonder if we could potentially have `Neo4jStellarGraph` check for a constrain on the ID (and maybe emit a warning if it doesn't exist)?

Reply:
That's a good idea, yeah that should be possible with `CALL db.constraints`

Reply:
There's now a bit of regex to find the correct constraint.. not 100% sure how reliable that is but I guess in worst case scenario the user sees a warning that doesn't apply. Thoughts?